js/notice-setup-form.js

```diff
@@ -26,9 +37,14 @@ const onInputTitle = (evt) => {
 };
 
 const onInputPrice = (evt) => {
+  const minPrice = MinPricesForTypes[inputType.value.toUpperCase()];
```

Comment:
```suggestion
  const minPrice = MinPricesForTypes[evt.currentTarget.value.toUpperCase()];
```
Наверное так, как и в `onInputType` ?